cpp_template_pkg/package.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>cpp_template_pkg</name>
+  <version>0.0.0</version>
+  <description>A template packge used to demonstrate ROS2 GitHub Actions</description>
+  <maintainer email="ros-tooling@googlegroups.com">ROS Tooling Working Group</maintainer>
```

Comment:
remove and put FIXME here - we don't want to get pinged about failures of all the projects coming from this template.